model/execution/DeliveryExecutionList.php

```diff
@@ -128,6 +131,42 @@ private function createExecution($cachedData, $extraFields)
         return $execution;
     }
 
+    private function getAdjustedTime(string $deliveryExecutionId): ?int
+    {
+        $deliveryExecution = $this->getDeliveryExecutionManagerService()->getDeliveryExecutionById($deliveryExecutionId);
+        $testSession = $this->getTestSessionService()->getTestSession($deliveryExecution);
+        $adjustedTime = null;
+        if ($testSession) {
+            /** @var AdjustmentMap $adjustmentMap */
+            $adjustmentMap = $testSession->getTimer()->getAdjustmentMap();
```

Comment:
Instead of using adjustmentMap directly (imlpementation level) I would recommend to create a new method in TimerAdjustmentService and use it via this service.

Reply:
done